### PR TITLE
Add basic DOS flows

### PR DESCRIPTION
**Pull request by robyoung**
For digital-specialists and digital-outcomes.

### CODE REVIEW

**Comment by quis** [comment]
:100: :tada:

File: frameworks/digital-outcomes-and-specialists/questions/services/outcomesApproachQuestion.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+type: text
+question: A question for outcomes
+depends:
+  - "on": lot
+    being: ["digital-outcomes"]
```
Comment (quis) [inline frameworks/digital-outcomes-and-specialists/questions/services/outcomesApproachQuestion.yml]: We generally use YAML-onic syntax for arrays, even if they only have one item, eg

``` YAML
depends:
  on: lot
  being:
    - digital-outcomes
```

File: frameworks/digital-outcomes-and-specialists/manifests/edit_submission.yml [hunk 2]
```diff
@@ -1,5 +1,63 @@
...
+    - technicalArchitectLocations
+
+- name: Delivery manager
+  editable: True
+  questions:
+    - deliveryManagerDayRate
+    - deliveryManagerLocations
+
+- name: Product manager
+  editbale: True
```
Comment (quis) [inline frameworks/digital-outcomes-and-specialists/manifests/edit_submission.yml]: `editable`

File: frameworks/digital-outcomes-and-specialists/questions/services/businessAnalystLocations.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+hint: ''
```
Comment (quis) [inline frameworks/digital-outcomes-and-specialists/questions/services/businessAnalystLocations.yml]: Don't think it'll cause any problems but it's sort of unclear what the intention of a blank `hint` is.

**Comment by quis** [comment]
Other than my comments, code looks good. Do @ralph-hawkins or @superroz need to review this to make sure that nothing's been omitted?

**Comment by ralph-hawkins** [comment]
In the way we have ```hint comprising of further information about pricing``` do we need ```hint comprising of further information about role```? cc @superroz and Sarah (I'll speak to her)

**Comment by quis** [comment]
- fixed up my own comments
- discussed with @ralph-hawkins, we're fine with placeholder content for the moment

:+1:

**Pull request merged**